plugin/trino-redshift/pom.xml

```diff
@@ -17,16 +17,6 @@
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
     </properties>
 
-    <repositories>
```

Comment:
Why this is no longer needed?

Reply:
Since v2.0.0.3 jars are published on maven central: https://mvnrepository.com/artifact/com.amazon.redshift/redshift-jdbc42

Reply:
very minor nit : Maybe as a separate commit ?

Reply:
Yeah, makes sense @Praveen2112